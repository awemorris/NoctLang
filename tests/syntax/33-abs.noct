func main() {
    var a = -123;
    print(abs(a));
    
    var b = 123;
    print(abs(b));

    var c = -1.23;
    print(abs(c));

    var d = 1.23;
    print(abs(d));
}
