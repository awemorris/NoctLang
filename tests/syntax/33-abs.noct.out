123
123
1.230000
1.230000
